internal/credential/vault/repository_credentials.go

```diff
@@ -38,6 +41,16 @@ func (r *Repository) Issue(ctx context.Context, sessionId string, requests []cre
 		if err != nil {
 			return nil, err
 		}
+
+		period, err := parseutil.ParseDurationSecond(cred.getExpiration())
```

Comment:
Do we need to parse this, can we not just do `if cred.getExpiration() < r.scheduler.GetRunJobsInterval()`

Reply:
good point